### PR TITLE
Update dlp-microsoft-teams.md

### PR DESCRIPTION
**Pull request by DanStevensonMSFT**
added an important note regarding the risky loophole around activity notifications

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/dlp-microsoft-teams.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ To learn more about licensing requirements, see [Microsoft 365 Tenant-Level Serv
 
 ## Overview of DLP for Microsoft Teams
 
-Recently, [data loss prevention](data-loss-prevention-policies.md) (DLP) capabilities were extended to include Microsoft Teams chat and channel messages, **including private channel messages**.
+Recently, [data loss prevention](data-loss-prevention-policies.md) (DLP) capabilities were extended to include Microsoft Teams chat and channel messages, **including private channel messages**. 
+
+> [!IMPORTANT]
+> DLP currently applies only to the actual messages in the chat or channel thread. Activity notifications -- which include a short message preview and appear based on a user's notification settings -- are **not** included in Teams DLP at this time. Any sensitive information present in the part of the message that appears in the preview will remain visible in the notification even after the DLP policy has been applied and removed sensitive inforamtion the message itself.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-microsoft-teams.md:32]:
```suggestion
> DLP currently applies only to the actual messages in the chat or channel thread. Activity notifications—which include a short message preview and appear based on a user's notification settings—are _not_ included in Teams DLP at this time. Any sensitive information present in the part of the message that appears in the preview will remain visible in the notification even after the DLP policy has been applied and removed sensitive information from the message itself.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@DanStevensonMSFT Please update with the feedback. Thanks

**Comment by yogkumgit** [comment]
@DanStevensonMSFT Gentle reminder

**Pull request merged**